### PR TITLE
PBX-14 : 4.3 vonix pbx 14 2

### PR DESCRIPTION
**Pull request by khandu-nehra-sprbrk**
PBX-14 : 4.3 vonix pbx 14 2

### CODE REVIEW

File: make/deps.mk [hunk 1]
```diff
@@ -1,4 +1,5 @@
-DEPS ?= amqp_client \
```
Comment (cking-vonix) [inline make/deps.mk]: I see a commit message that says you revert the changes to deps.mk but it doesn't say why these changes are made?  PBX-14 was already merged and completed.

File: core/sup/Makefile [hunk 2]
```diff
@@ -4,7 +4,7 @@ PROJECT = sup
 ESCRIPT = sup
 CLEAN_MOAR = clean-escript
 
-all: compile $(ESCRIPT)
+all: compile #$(ESCRIPT)
```
Comment (cking-vonix) [inline core/sup/Makefile]: Why are we commenting out escript?   This builds the `sup` tool that is used, we absolutely need it.

File: make/deps.mk (hunk 1 above)
Comment (khandu-nehra-sprbrk) [reply make/deps.mk]: We are getting below error , When we build application on fresh machine

escript: exception error: no match of right hand side value undefined
  in function  tags_escript_escript1707830701828567_2:app_path/1 (./scripts/tags.escript, line 35)
  in call from tags_escript_escript1707830701828567_2:'-main/1-lc$^1/1-0-'/1 (./scripts/tags.escript, line 11)
  in call from tags_escript_escript1707830701828567_2:'-main/1-lc$^1/1-0-'/1 (./scripts/tags.escript, line 11)
  in call from tags_escript_escript1707830701828567_2:main/1 (./scripts/tags.escript, line 11)
  in call from escript:run/2 (escript.erl, line 758)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1 
  in call from init:do_boot/3 
make: * [Makefile:126: /home/user/kazoo/TAGS] Error 127

So we added dependencies back to make/deps.mk

File: core/sup/rebar.config [hunk 3]
```diff
@@ -1,5 +1,8 @@
+ {deps, [lager,jiffy,getopt,recon,zucchini,erlang_localtime
+,{inet_cidr, "1.0.2", {git, "https://github.com/2600hz/erlang-inet_cidr.git", {tag, "1.0.2"}}}
+]}.
 {lib_dirs, ["../../deps", "../../core"]}.
-{escript_incl_apps, [getopt, kazoo, kazoo_stdlib, kazoo_config, zucchini, lager]}.
```
Comment (cking-vonix) [inline core/sup/rebar.config:2]: `sup` is a command line interface for communicating with Kazoo, while getopt is a tool for parsing command line options.  Without it, sup will not work.

File: make/deps.mk (hunk 1 above)
Comment (cking-vonix) [reply make/deps.mk]: Understood, but do we need all of these dependencies or is there one specifically that is required?  `apns` and `fcm` are just for doing push notifications with Apple and Google respectively and I don't think will have any impact on this tool.

File: core/sup/rebar.config (hunk 3 above)
Comment (khandu-nehra-sprbrk) [reply core/sup/rebar.config:2]: While resolving sup build issue we moved this to deps. As dependency path is not resolving properly.
Now I tried adding back and build is working fine.
